11-if/exercises/04-vowel-or-cons/main.go

```diff
@@ -49,4 +45,18 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	l := len(os.Args) - 1
+
+	if l != 1 || len(os.Args[1]) != 1 {
+		fmt.Println("Give me a letter")
+		return
+	}
+
+	if strings.IndexAny(os.Args[1], "aeiou") == 0 {
```

Comment:
I think its better if we store the os.Args[1] into a variable and then compare it ... like letter = os.Args[1]

Reply:
I made the os.Args[1] variable into the var "args" and used this new variable instead of os.Args[1]